### ISSUE TITLE
Pierone does not store image correctly

### ISSUE THREAD

**Issue by kanuku** [zalando-stups/pierone#9]
It seems that pierone is omitting information from the image.

After uploading an image to the pierone registry the `docker inspect` command shows that many values are not stored correctly:
##### Another way to reproduce the problem

``` sh
# First pull Jenkins image and inspect it
docker pull jenkins  
docker inspect jenkins (save the result)

# Rename and push it to pierone
docker tag jenkins pierone/test/jenkins
docker push pierone/test/jenkins

# Delete any copy from local registry, do not untag it only, but completly purge it
docker rmi -f ImageId 
docker pull pierone/test/jenkins

## Analyse it and compare it with the first
docker inspect pierone/test/jenkins (save the result)

```

You can see the difference in Exposed ports and Volume definitions.

[Gist for before uploading the image](https://gist.github.com/kanuku/be5937345be00f6e40c1)  

[Gist for after uploading the image](https://gist.github.com/kanuku/f0765625be3138ff209f)

**Comment by hjacobs** [comment]
I could not reproduce this problem yet (also nobody else reported anything like this)

**Comment by dryewo** [comment]
I have this problem too.
Before pushing I have

```
"Volumes": {
            "/var/jenkins_home": {}
        },
```

then, after pushing, removing and pulling I have

```
"Volumes": {
            "var/jenkins_home": {}
        },
```

I've tried exactly the same scenario that @kanuku provides, and I've also tried to build my custom image based on `jenkins`, the "Volumes" property gets distorted in either case.

I insist, this is a bug :)

**Comment by s-soroosh** [comment]
Yea I had the same problem too.
@dryewo 
If in Dockerfile you mention "//var/jenkins_home" this problem will be solved.
Definitely it's just an interim solution and I am agree that this is a bug!

**Comment by hjacobs** [comment]
verified as bug

**Comment by hjacobs** [comment]
relates to https://github.com/zalando-stups/pierone/issues/24 too

**Comment by hjacobs** [comment]
fixed in release 0.6.0 https://github.com/zalando-stups/pierone/releases/tag/0.6.0
